homeassistant/components/smartenergy_goecharger/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "smartenergy_goecharger",
+  "name": "go-e Charger Cloud",
+  "documentation": "https://www.home-assistant.io/integrations/smartenergy_goecharger",
+  "iot_class": "cloud_polling",
+  "config_flow": true,
+  "dependencies": ["sensor"],
+  "codeowners": ["@openkfw"],
+  "requirements": ["smart-energy.goecharger-api==0.3.1"]
+}
```

Comment:
This will currently fail style checks. It needs to be in order of domain, name, then alphabetical. This is a recent CI change. not 100% sure if it is intentional